parachain/pallets/outbound-queue/runtime-api/src/lib.rs

```diff
@@ -2,11 +2,14 @@
 // SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use snowbridge_core::outbound::{Message, OutboundFee};
 use snowbridge_outbound_queue_merkle_tree::MerkleProof;
 
 sp_api::decl_runtime_apis! {
 	pub trait OutboundQueueApi
 	{
 		fn prove_message(leaf_index: u64) -> Option<MerkleProof>;
+
+		fn estimate_fee(message: Message) -> Option<OutboundFee<u128>>;
```

Comment:
Prefer this name instead: `calculate_fee`.

Reply:
[Rename to calculate_fee](https://github.com/Snowfork/snowbridge/pull/975/commits/8d6ec9d2774444a4ae356dbb9803ca67eb186475)